subsys/bluetooth/controller/include/ll.h

```diff
@@ -8,6 +8,8 @@
 #ifndef _LL_H_
 #define _LL_H_
 
+#include <bluetooth/hci.h>
```

Comment:
It is not Zephyr Bluetooth API, similar patch set to include header files was rejected before.